rnd/autogpt_server/migrations/20240724150133_node_input_unique_constraint/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
```

Comment:
I believe you need to run migrations on the Postgres version as well now. cc. @aarushik93

Reply:
I don't think postgres migration is maintained 
@aarushik93 let me if this is wrong, otherwise I can followup in a separate PR